main.py

```diff
@@ -14,7 +15,7 @@
 FAIR_ODDS_PARAMETER = 0.5
 
 
-def get_match_data(odds_probability_type):
+def get_match_data(odds_probability_type: str) -> list:
```

Comment:
Use `typing.List` instead, preferably with the type that the list contains (i.e. `List[str]`)